apps/condo/domains/ticket/access/Ticket.js

```diff
@@ -42,7 +47,10 @@ async function canManageTickets ({ authentication: { item: user }, operation, it
         if (!property) {
             return false
         }
-
+        // TODO(zuch): need to check organization and property
```

Comment:
?)

Reply:
In fact we do not need to check this on create. If user do not have access to property or organization - he can not add such ticket. There will be connection error